react-delegationdashboard/src/components/Layout/APRCalculation.ts

```diff
@@ -54,9 +54,7 @@ const calculateAPR = ({
   const allActiveNodes = blsKeys.filter(key => key.asString === 'staked').length;
   const validatorBaseStake = allActiveNodes * stakePerNode;
   const validatorTotalStake = parseInt(denominateValue(totalActiveStake));
```

Comment:
For validatorTopUpReward we should use the validatorTopUpStake not the validatorTotalStake